input_dictionary.json

```diff
@@ -461,6 +459,130 @@
           "options": ["Constant", "Logaritmic", "Square root", "Cube root"]
         }
       ]
-    }
-  ]
-}
+    },
+		{
+		"estimator_id": "PKEstimator",
+		"algorithm_id": "PKAlgorithm",
+		"display_label": "PKEstimator",
+		"landing_page_content": " # Permuted Kernel Estimator\n\nThis project provides an estimator for the hardness of the permuted kernel problem. This problem is defined as follows: \n\nGiven two matrices $\\mathbf{A} \\in \\; \\mathbb{F}_{q}^{m\\times n}$ and $\\mathbf{V} \\in \\; \\mathbb{F}_{q}^{\\ell \\times n}$, the permuted kernel problem asks to find a permutation $\\mathbf{P} \\in \\mathbb{F}_{q}^{m\\times n}$ such that $\\mathbf{A}(\\mathbf{V} \\mathbf{P})^\\top = 0$\n\nMore details on the theoretical foundations of the estimator can be found in the corresponding papers: \n\n*[[SBC22]](https://eprint.iacr.org/2022/1749.pdf) Paolo Santini, Marco Baldi, and Franco Chiaraluce. Computational hardness of the permuted kernel and subcode equivalence problems.*",
+		"problem_parameters": [
+			{
+				"id": "n",
+				"type": "number",
+				"display_label": "Number of columns",
+				"placeholder": "Insert parameter",
+				"tooltip": "Number of columns of the input matrix"
+			},
+          	{
+				"id": "m",
+				"type": "number",
+				"display_label": "Number of rows",
+				"placeholder": "Insert parameter",
+				"tooltip": "Number of rows of the input matrix"
+			},
+           {
+				"id": "q",
+				"type": "number",
+				"display_label": "Field size",
+				"placeholder": "Insert parameter",
+                "tooltip": "An integer of the form p^x for some integer x, where p is prime number.  This value indicates the number of elements in the underlying field"
```

Comment:
actually for PKP I think the q should be prime, defined over extension fields it might be less secure

Reply:
I mean the algorithms still work in that case, but it might give the impression we claim that in this case those algorithms still yield a good estimate for the general hardness (meaning they are still the best, which is likely not the case)

Reply:
Now it says in the tooltip it is a prime number, is it enough?